napari/layers/intensity_mixin.py

```diff
@@ -121,7 +125,9 @@ def gamma(self):
 
     @gamma.setter
     def gamma(self, value):
+        self.events.gamma(value=value)
```

Comment:
this is where we could just use `self.events.gamma(value)` right (and other places above)?   are we going to try to use that form?

Reply:
yes, i'll change that - not sure how that one slipped by!! not needed anymore